apollo-portal/src/main/resources/static/scripts/controller/AuditLogMenuController.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+audit_log_menu_module.controller('AuditLogMenuController',
+    ['$scope', '$window', '$translate', 'toastr', 'AppService', 'AppUtil', 'EventManager', 'AuditLogService',
+      auditLogMenuController]
+);
+
+function auditLogMenuController($scope, $window, $translate, toastr, AppService, AppUtil, EventManager, AuditLogService) {
+
+      $scope.auditLogList = [];
+      $scope.goToTraceDetailsPage = goToTraceDetailsPage;
+      $scope.searchByOpNameAndDate = searchByOpNameAndDate;
+      $scope.getMoreAuditLogs = getMoreAuditLogs;
+
+      $scope.page = 0;
+      var PAGE_SIZE = 10;
```

Comment:
Add page start number and page size as javadoc in java controller or service

Reply:
added